uaclient/entitlements/fips.py

```
@@ -173,7 +173,9 @@ def _allow_fips_on_cloud_instance(
     def static_affordances(self) -> "Tuple[StaticAffordance, ...]":
         # Use a lambda so we can mock util.is_container in tests
         cloud_titles = {"azure": "an Azure", "gce": "a GCP"}
-        cloud_id = get_cloud_type() or ""
+        cloud_id, _error = get_cloud_type()
```

Same thing here about ignoring the error var